pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -227,7 +235,17 @@ func (o *InstallOptions) copyContent() error {
 	if err := os.MkdirAll(o.PodManifestDir, 0755); err != nil {
 		return err
 	}
+
+	if o.PodMutationFn != nil {
+		glog.Infof("mutating static pod")
```

Comment:
this sounds dangerous, like a warning. "Customizing" or "Adapting" is a better word maybe.

Reply:
> this sounds dangerous, like a warning. "Customizing" or "Adapting" is a better word maybe.

ok